bagitobjecttransfer/recordtransfer/templates/includes/pagination.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+{% load query_params %}
+<div class="pagination">
+    {% if page_obj.has_previous %}
+        <a href="?{% update_query_params pagination_name page_obj.previous_page_number %}"
+           class="btn-arrow btn-arrow-left">{% trans "Previous" %}</a>
+    {% else %}
+        <a class="btn-arrow btn-arrow-left disabled" aria-disabled="true">{% trans "Previous" %}</a>
+    {% endif %}
+    <span>{% trans "Page" %} {{ page_obj.number }} {% trans "of" %} {{ page_obj.paginator.num_pages }}</span>
```

Comment:
A translated version of this page may not have the page number in the same form "page 1 of 3". In another language, it might be more natural to write "1 of 3 pages", for example.

In these cases when you have to use variables inside a translatable block, you can use `blocktrans` with `with`.

```suggestion
    <span>
        {% blocktrans with curr_page=page_obj.number total_pages=page_obj.paginator.num_pages %}
        Page {{ curr_page }} of {{ total_pages }}
        {% endblocktrans %}
    </span>
```

Reply:
That's an interesting case I didn't consider, thanks for pointing it out!

Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/280/commits/35e11ad778671b75c2f894b1639882caea2d0def